firefox-ios/Client/TabManagement/Tab.swift

```diff
@@ -1109,7 +1119,7 @@ class TabWebView: WKWebView, MenuHelperWebViewInterface, ThemeApplicable {
     override func evaluateJavaScript(_ javaScriptString: String, completionHandler: ((Any?, Error?) -> Void)? = nil) {
```

Comment:
I believe that the `evaluateJavaScript` overriden methods should be deleted since they don't do anything apart from calling `super`.